mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/store/NoopStorageProvider.java

```diff
@@ -16,150 +16,49 @@
 
 package io.mantisrx.server.master.store;
 
-import io.mantisrx.master.resourcecluster.DisableTaskExecutorsRequest;
-import io.mantisrx.server.master.resourcecluster.ClusterID;
-import io.mantisrx.server.master.resourcecluster.TaskExecutorID;
-import io.mantisrx.server.master.resourcecluster.TaskExecutorRegistration;
-import io.mantisrx.shaded.com.google.common.collect.ImmutableList;
-import java.io.IOException;
-import java.util.ArrayList;
 import java.util.List;
-import rx.Observable;
+import java.util.Map;
 
 
 public class NoopStorageProvider implements MantisStorageProvider {
```

Comment:
I would maybe define this in the MantisStorageProvider itself and then have a static method MantisStorageProvider::noop() to get this instance.

Reply:
done, good suggestion.

Reply:
Not used anymore right?

Reply:
ah yes, seems to have stuck around because of some recent rebase. thx!